### ISSUE TITLE
Craft update failure 3.7.16 => 3.7.17.1

### ISSUE THREAD

**Issue by JordanMed** [craftcms/cms#9990]
### Description

I am getting the following error when updating craft CMS

```
Response: {"error":"User is not permitted to perform this action","exception":"yii\\web\\ForbiddenHttpException","file":"/Users/usr/Sites/web/vendor/craftcms/cms/src/web/Controller.php","line":280,"trace":[{"file":"/Users/usr/Sites/web/vendor/craftcms/cms/src/controllers/UpdaterController.php","line":46,"function":"requirePermission","class":"craft\\web\\Controller","type":"->"},{"file":"/Users/usr/Sites/web/vendor/yiisoft/yii2/base/Controller.php","line":179,"function":"beforeAction","class":"craft\\controllers\\UpdaterController","type":"->"},{"file":"/Users/usr/Sites/web/vendor/yiisoft/yii2/base/Module.php","line":534,"function":"runAction","class":"yii\\base\\Controller","type":"->"},{"file":"/Users/usr/Sites/web/vendor/craftcms/cms/src/web/Application.php","line":277,"function":"runAction","class":"yii\\base\\Module","type":"->"},{"file":"/Users/usr/Sites/web/vendor/craftcms/cms/src/web/Application.php","line":586,"function":"runAction","class":"craft\\web\\Application","type":"->"},{"file":"/Users/usr/Sites/web/vendor/craftcms/cms/src/web/Application.php","line":256,"function":"_processActionRequest","class":"craft\\web\\Application","type":"->"},{"file":"/Users/usr/Sites/web/vendor/yiisoft/yii2/base/Application.php","line":392,"function":"handleRequest","class":"craft\\web\\Application","type":"->"},{"file":"/Users/usr/Sites/web/web/index.php","line":21,"function":"run","class":"yii\\base\\Application","type":"->"}]}
```

### Steps to reproduce

1. update from 3.7.16 to 3.7.17.1
2.

### Additional info

- Craft version: 3.7.16
- PHP version: 7.4.16
- Database driver & version: 5.7.32
- Plugins & versions:

**Comment by brandonkelly** [comment]
Craft 3.7.19 is out with that fix. Note that it is still expected that you will be logged out after updating, but you at least won’t get that error.

**Comment by tommers-walker** [comment]
Having the same issue, stuck in an endless 403 loop. No joy in incognito or a different browser either

**Comment by ben-sharp** [comment]
I also have the same issue with endless 403 in any browser/fresh sessions on 3.7.19

**Comment by tommers-walker** [comment]
Think this might have had something to do with a failed migration, managed to fix it with `php craft migrate/all`

**Comment by ben-sharp** [comment]
I was able to downgrade to 3.7.17 then upgrade to 3.7.19 without hitting the issue

**Comment by zizther** [comment]
I also downgraded to 3.7.17 and it worked fine, I was then able to update to 3.7.19.

**Comment by brandonkelly** [comment]
@zizther @tommers-walker @ben-sharp Do any of you have a database backup from 3.7.18?

**Comment by ben-sharp** [comment]
> @zizther @tommers-walker @ben-sharp Do any of you have a database backup from 3.7.18?

no sorry!

**Comment by zizther** [comment]
@brandonkelly Unfortunately not in this case.

I am facing another odd issue which seems like it is related to the issue above.
I have been doing some updates on a site locally, I updated from 3.7.19 to 3.7.19.1 (This site was previously updated from 3.7.18). I have noticed a major issue where I get a 503 error on the frontend when logged out the CP, when I login to the CP I am able to access the FE fine.
If I log out I am not able to access the FE as it shows the same 503 error again.

It appears as if a user needs to be logged into the CP to access the site, even though nothing has been setup to make this the case in the CP or general config. Downgrading to 3.7.17 or 3.7.16 did not solve this issue.

PHP 8.0.8
Apache 2.4.48
opCache disabled
MySQL 5.7.34
Craft 3.7.19.1
